CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## 0.7.10dev
 
-* [Feature] Modified `TableDescription` to add styling, generate messages and format the calculated outputs (#459)
 * [Feature] Support flexible spacing `myvar=<<` operator ([#525](https://github.com/ploomber/jupysql/issues/525))
+* [Feature] Moved `%sqlrender` feature to `%sqlcmd snippets` (#647)
+* [Feature] Modified `TableDescription` to add styling, generate messages and format the calculated outputs (#459)
```

Comment:
Is this a part of this PR? If not please revert

Reply:
Seems like the order changed when I merge a conflict manually. It's now fixed.